tools/autograd/derivatives.yaml

```diff
@@ -1465,8 +1465,9 @@
 - name: nansum.dim_IntList(Tensor self, int[1] dim, bool keepdim=False, *, ScalarType? dtype=None) -> Tensor
   self: nansum_backward(grad.to(self.scalar_type()), self, dim, keepdim)
 
-- name: _svd_helper(Tensor self, bool some, bool compute_uv) -> (Tensor U, Tensor S, Tensor V)
-  self: svd_backward(grads, self, some, compute_uv, U, S, V)
+# We never call _linalg_svd with compute_uv=False in an autograd context, so we don't even consider it here
```

Comment:
Let's play it safe and include at least a TORCH_INTERNAL_ASSERT. If it ever happens users would be asked to submit a bug report.

Reply:
So, I was going to do it, but we don't import the those asserts to the FunctionsManual.cpp. I am a bit concerned that adding another dependency might break mobile, as it did for https://github.com/pytorch/pytorch/pull/69824 
I'd say it's fine to leave it as it is, as if `U` or `Vh` were not computed then the function is going to fail hard anyway, so we'd get an issue if that happens.

Reply:
All that I suggest is to pass `compute_uv` to `svd_backward` and check that it's never `false`.

Reply:
What would be executed here?
```py
import torch
a = torch.randn(3, 3, requires_grad=True)
_, s, _ = torch._linalg_svd(a, compute_uv=False)
s.backward(torch.ones_like(s))
```
Is `_linalg_svd` bound to `torch.` module?

Reply:
Are functions prepended with an underscore accessible from the Python API? I'd expect they are not.

Reply:
Ugh, they are indeed exposed to the API. What is it with this @mruberry @albanD ?